extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -210,7 +211,16 @@ private Containerizer createContainerizer(ContainerImageConfig containerImageCon
                     containerImageConfig.password);
             containerizer = Containerizer.to(registryImage);
         } else {
-            containerizer = Containerizer.to(DockerDaemonImage.named(imageReference));
+            DockerDaemonImage dockerDaemonImage = DockerDaemonImage.named(imageReference);
+            Optional<String> dockerConfigExecutableName = ConfigProvider.getConfig()
```

Comment:
Is this configuration available in another config class?

Reply:
It is available in `io.quarkus.container.image.docker.deployment.DockerConfig` which is another extension (and we thus can't reuse it).
We do want users to be able to use the property however as was done in https://github.com/quarkusio/quarkus/pull/20742

Reply:
Understood, I think the only concern I have here is when this property is renamed or deprecated/replaced.

Reply:
Yeah, that will require some vigilance on our part :)